x/nexus/keeper/migrate.go

```diff
@@ -4,10 +4,25 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// GetMigrationHandler returns the handler that performs in-place store migrations from v0.14 to v0.15. The
+// GetMigrationHandler returns the handler that performs in-place store migrations from v0.24 to v0.25. The
 // migration includes:
+//   - migrate maintainer states
 func GetMigrationHandler(k Keeper) func(ctx sdk.Context) error {
 	return func(ctx sdk.Context) error {
+		migrateMaintainerStates(ctx, k)
+
 		return nil
 	}
 }
+
+func migrateMaintainerStates(ctx sdk.Context, k Keeper) {
+	for _, chainState := range k.getChainStates(ctx) {
+		for _, maintainerState := range chainState.MaintainerStates {
```

Comment:
Add a log to print the number of maintainers being migrated, just for sanity?

Reply:
not sure why is it necessary since there's the test, but ok